website/internal/views/__init__.py

```diff
@@ -48,5 +49,25 @@ def themes():
 
 @internal_site.route("/graph")
 def graph():
-    data = data_interface.get_user_graph_data(utilities.session.get_active_user()['user_id'])
-    return render_template("internal/graph.html", data=data)
+    return render_template("internal/graph.html")
+
+@internal_site.route('/graph/data')
+def graph_data():
+    si = StringIO()
+    cw = csv.writer(si)
+    #data = data_interface.get_user_graph_data(utilities.session.get_active_user()['user_id'])
```

Comment:
I modified the API so you can use the actual data (that data is random, but that's for the backend team to fix). You can merge my pull request for that.